### PR TITLE
Feature/add theme VB-14

### PR DESCRIPTION
**Pull request by VladimirBludov**

### CODE REVIEW

**Review by Vitalii4as — CHANGES_REQUESTED** [review]
Wait when #14 is merged. Use colors from there

File: packages/mobile/src/config/navigation-theme.config.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { ColorPalette } from '@vse-bude/shared';
+import { DefaultTheme, DarkTheme } from '@react-navigation/native';
+
+const COMMON_COLORS = {
+  accent: ColorPalette.YELLOW_100,
+  organ: ColorPalette.GREEN_200,
+  card: ColorPalette.GRAY_100,
+  inputBg: ColorPalette.GRAY_100,
+  dropdownBg: ColorPalette.GRAY_100,
+  circleBg: ColorPalette.GRAY_100,
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/config/navigation-theme.config.ts:10]: We may combine it under some common name.

File: packages/mobile/src/hooks/hooks.ts [hunk 2]
```diff
@@ -4,3 +4,4 @@ export { useController as useFormControl } from 'react-hook-form';
 export { useAppDispatch } from './use-app-dispatch/use-app-dispatch.hook';
 export { useAppSelector } from './use-app-selector/use-app-selector.hook';
 export { useAppForm } from './use-app-form/use-app-form.hook';
+export { useColorScheme } from 'react-native';
```
[the remark this answers is not in the document]
Comment (VladimirBludov) [reply packages/mobile/src/hooks/hooks.ts]: corrected

File: packages/mobile/src/config/navigation-theme.config.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  line: ColorPalette.GRAY_200,
+  placeholder: ColorPalette.BLACK_100,
+  subtitle: ColorPalette.GRAY_300,
+  buttonTextPrimary: ColorPalette.WHITE_100,
+  buttonTextSecondary: ColorPalette.GREEN_200,
+  buttonBgDark: ColorPalette.GREEN_200,
```
[the remark this answers is not in the document]
Comment (VladimirBludov) [reply packages/mobile/src/config/navigation-theme.config.ts]: corrected

File: packages/mobile/src/config/navigation-theme.config.ts (hunk 1 above)
Comment (VladimirBludov) [reply packages/mobile/src/config/navigation-theme.config.ts]: corrected

**Review by AleksandrTermenzhy — APPROVED** [review]

**Review by Vitalii4as — APPROVED** [review]

**Pull request merged**